src/commands/dataconnect-sdk-generate.ts

```diff
@@ -8,9 +8,11 @@ import { load } from "../dataconnect/load";
 import { readFirebaseJson } from "../dataconnect/fileUtils";
 import { logger } from "../logger";
 
+type GenerateOptions = Options & { watch?: boolean };
```

Comment:
While we are at this. It would nice if `sdk:generate` also takes optional `[serviceId] [connectorId]` like other commands.

```
  dataconnect:services:list                                      list all deployed services in your Firebase 
                                                                 project
  dataconnect:sql:diff [serviceId]                               displays the differences between  a local 
                                                                 DataConnect schema and your CloudSQL database's 
                                                                 current schema
  dataconnect:sql:migrate [options] [serviceId]                  migrates your CloudSQL database's schema to 
                                                                 match your local DataConnect schema
  dataconnect:sql:grant [options] [serviceId]                    Grants the SQL role <role> to the provided user 
                                                                 or service account <email>.
  dataconnect:sdk:generate                                       generates typed SDKs for your Data Connect 
                                                                 connectors
```